### PR TITLE
Update no-NB.json

### PR DESCRIPTION
**Pull request by haraldgroven**
Added new missing translations to locale `no-NB` in order to bring Norwegian UI back to latest version. 

Notes
The tool `translation-helper` skipped and didn't write several strings, eg. Sweden's regions. 
I haven't checked if URLs in locale point to correct (current) URLs.

### CODE REVIEW

File: web/public/locales/no-NB.json [hunk 1]
```diff
@@ -705,7 +752,7 @@
       "zoneName": "Guyana"
     },
     "HK": {
-      "zoneName": "Hong Kong"
+      "zoneName": "Hong-Kong"
```
Comment (VIKTORVAV99) [inline web/public/locales/no-NB.json:755]:
```suggestion
      "zoneName": "Hongkong"
```

At least according to Wikipedia 😅

**Review by VIKTORVAV99 — COMMENTED** [review]
Just one thing to look at as far as I can tell.

File: web/public/locales/no-NB.json (hunk 1 above)
Comment (madsnedergaard) [reply web/public/locales/no-NB.json:755]: Seems like this was not changed, did you perhaps forget to push it or was the dashed version actually correct? :)

**Review by madsnedergaard — APPROVED** [review]
LGTM! :)

**Pull request merged**